src/components/Panel/Panel.tsx

```diff
@@ -71,6 +85,10 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
 
     let pendingCommandBarContent = '';
 
+    if (!isOpen) {
```

Comment:
I don't think that changes your PR here... but this will need to be fixed in a later checkin.